get_golang_versions.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# -*- mode: bash; -*-
+
+# File: get_golang.sh
+# Copyright (C) 2019 Gravitational Inc.
+# Description: Find 3 latest major Golang releases
+
+# set -o xtrace
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function decrement_version() {
+    local VERSION="$1"
+
+    local DECREMENTED_VERSION=
+    if [[ "$VERSION" =~ .*\..* ]]; then
+        DECREMENTED_VERSION="${VERSION%.*}.$((${VERSION##*.}-1))"
+    else
+        DECREMENTED_VERSION="$((${VERSION##*.}-1))"
+    fi
+
+    echo "$DECREMENTED_VERSION"
+}
+
+function find_latest_minor_release() {
+    local RELEASES=("$@")
+    local MAJOR_RELEASE="${RELEASES[-1]}"
+
+    for version in ${RELEASES[@]}; do
+        if [[ "$version" =~ ^"$MAJOR_RELEASE" ]];
+        then
+            echo $version
+            break
+        fi
+    done
+}
+
+readarray releases_list <<< $(wget -qO- https://golang.org/dl/ | grep -oP 'https:\/\/dl\.google\.com\/go\/go([0-9\.]+)\.linux-amd64\.tar\.gz' | head -n 30 | sed -nre 's/^[^0-9]*(([0-9]+\.)*[0-9]+).*/\1/p' | uniq )
```

Comment:
I don't know how likely it is, but uniq only considers adjacent list items for uniqueness. If removing duplicates is really important, this may need to be more robust.

Reply:
It is absolutely not important here, I need to remove only one duplicate for the latest release (i.e 1.13 now and next download link also points on the same 1.13 release).